ansible/group_vars/all/defaults.yaml

```diff
@@ -6,6 +6,8 @@ kubernetes_cni_version: "0.9.1"
 etcd_version: "3.4.13-0"
 coredns_version: "1.8.0"
 pause_image_version: "3.4.1"
```

Comment:
Should we make this a list of `pause_image_versions` and pull all the versions in the list?